pkg/config/config.go

```diff
@@ -84,6 +84,13 @@ func LoadConf(bytes []byte) (*sriovtypes.NetConf, error) {
 		return nil, fmt.Errorf("LoadConf(): invalid link_state value: %s", n.LinkState)
 	}
 
+	// validate vlan_trunk; if vlan is given vlan_trunk will be ignored
```

Comment:
Do we want to have a condition check when both vlan and vlan_trunk are configured, and return explicit error?
Currently, when vlan is configured, vlan_trunk will be ignored silently.

Reply:
Yes, we could add explicit error if both configuration is given.